src/main/webapp/scripts/login.js

```diff
@@ -0,0 +1,12 @@
+async function loadLogInURL() {
```

Comment:
Could give a clarifying comment to thins function?

Reply:
Yes, if the user isn't logged in, they will be redirected to the login URL. If they are logged in, they will be sent to the readfile.html page.